spark-3.3/clickhouse-spark-it/src/test/scala/org/apache/spark/sql/clickhouse/single/WriteDistributionAndOrderingSuite.scala

```diff
@@ -78,7 +85,7 @@ class WriteDistributionAndOrderingSuite extends SparkClickHouseSingleTest {
     WRITE_REPARTITION_BY_PARTITION.key -> repartitionByPartition.toString,
     WRITE_LOCAL_SORT_BY_KEY.key -> localSortByKey.toString
   ) {
-    if (!ignoreUnsupportedTransform && repartitionByPartition) {
+    if (!ignoreUnsupportedTransform && repartitionByPartition && !isCloud) {
```

Comment:
why does it not work on clickhouse cloud?

Reply:
It does not throw  the exception on Cloud that we are expecting on the Cloud Service